lib/shared/bucketing-assembly-script/assembly/types/dvcEvent.ts

```diff
@@ -139,8 +139,8 @@ export class DVCRequestEvent extends JSON.Value {
         if (this.target) json.set('target', this.target)
         if (this.customType) json.set('customType', this.customType)
         json.set('user_id', this.user_id)
-        json.set('date', this.date)
-        json.set('clientDate', this.clientDate)
+        json.set('date', this.date.toISOString())
```

Comment:
why the switch to date strings?

Reply:
that's what the API docs have: https://events.devcycle.com/docs#/Events/EventsController_submitEventsBatchV1